libs/blocks/tabs/tabs.css

```diff
@@ -538,12 +598,10 @@
     margin-bottom: 0;
   }
 
-  .tabs.stacked-mobile div[role="tablist"] button[aria-selected="true"] {
-    background-color: #dadada;
-  }
-  
+  .tabs.stacked-mobile div[role="tablist"] button[aria-selected="true"],
   .tabs.stacked-mobile.dark div[role="tablist"] button[aria-selected="true"] {
```

Comment:
Can't these 2 selectors be consolidated? Wouldn't the first one also apply to the `.dark` variant or are there specificity issues?